packages/vscode-extension/README.md

```diff
@@ -1,36 +1,66 @@
-# Microsoft 365 Agents Toolkit for Visual Studio Code
+# Microsoft 365 Agents Toolkit - an evolution of Teams Toolkit
 
 ## What is Microsoft 365 Agents Toolkit?
 
-The Microsoft 365 Agents Toolkit helps developers create and deploy apps with integrated identity, access to cloud storage, data from [Microsoft Graph](https://docs.microsoft.com/graph/teams-concept-overview), and other services in [Azure](https://docs.microsoft.com/microsoftteams/platform/build-your-first-app/build-bot) and [Microsoft 365](https://docs.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant) with a "zero-configuration" approach to the developer experience.
+The Microsoft 365 Agents Toolkit, an evolution of Teams Toolkit, is designed to help developers create and deploy agents or apps for multiple Microsoft 365 platforms including Microsoft 365 Copilot, Microsoft Teams and Microsoft 365. It can significantly ease your development life by providing integrated [Microsoft 365](https://docs.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant) identity, cloud storage access, data from [Microsoft Graph](https://docs.microsoft.com/graph/teams-concept-overview), and other services in [Azure](https://docs.microsoft.com/microsoftteams/platform/build-your-first-app/build-bot) with a "zero-configuration" approach.
```

Comment:
please hyperlink the "an evolution of Teams Toolkit" using the aka.ms link I posted to the Teams chat. That link is premade for all future overview on M365 Agents Toolkit.